### DISCUSSION TITLE
Can Open Shell interfere with peripheral installation?

### DISCUSSION THREAD

**Discussion by MickM9** [Open-Shell/Open-Shell-Menu#1253] [Q&A] [answered]
I'm setting up a newly built computer that runs Win10.  My previous rig ran Win7pro.  When I installed my old HP printer it went in and mostly worked but needed cleaning and adjustments.  Alas, when it installed it didn't set up the start menu folder with links to the utilities that are needed to make those adjustments or cleaning.   So I'm left to wonder - do those printer utilities simply not work, or not even load, into Win10?  Or did Open Shell somehow prevent the creation of the menu folder that I would use to access those utilities?

I'm guessing that the old printer just won't install properly or work properly in Win10, especially since I couldn't find those utilities in the program folder, but I hesitate to buy a new printer if it could be an installation issue with the old one that might happen with a new one too.  So I'm asking if someone here who's smarter or more knowledgeable than me might know the answer, or at least offer a clue.

Failing a definite answer, is there a way to temporarily inactivate the Open Shell menu, do the experiment and turn it back on?  I looked at the settings and didn't see anything that looked such an option.

I installed all the rest of my hardware and software before I installed Open Shell, so I can't say if this issue is unique to the printer.  I had some software that just wouldn't install or run in 10 but I expected that with some very old programs.  Nothing else did this sort of install but not completely, and sort of work but not really, like the printer.

**Comment by ge0rdi** [comment, marked as answer]
It is very unlikely that `Open-Shell` would interfere with application installation.
You can try to reinstall your printer software. Maybe that would help?

If you want to disable `Open-Shell` to test whether the issue could be related to it, the best would be to uninstall it temporarily (and reboot to be sure nothing left running anywhere).
All your settings will be kept on disk, so once you install it again everything should work as before.

**Reply by MickM9** [reply]
Yeah, that's what I thought.  The printer is plug & play and came with no software other than any built in so it ran when the printer was plugged in.  I wondered if trouble creating the start menu items might have somehow put the brakes on the installation process before it finished.  But the more I think about it, the less I think so.  I think my old wood-burning printer just won't work properly with my new operating system.

I'm just going to get a new printer.  I'm about to start shopping for it right now.

So we can consider this thread closed.

Thanks for your reply!  I appreciate it.

**Comment by ge0rdi** [comment]
> I think my old wood-burning printer just won't work properly with my new operating system.

You should check HP site for information on OS compatibility.
It would be no surprise that old HW is not supported in modern OS.
